### PR TITLE
Add info on how to to add ZSH plugin (CLI)

### PR DESCRIPTION
**Pull request by randavidovitz**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.7 Features: set Milestone to `1.7` and Base Branch to `release-1.7`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3787)

### CODE REVIEW

**Comment by randavidovitz** [comment]
i have associated it and it doesn't accept ?

File: docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -129,6 +129,14 @@ echo "source <(kubectl completion bash)" >> ~/.bash_profile
 Please note that this only appears to work currently if you install using `brew install kubectl`,
 and not if you downloaded kubectl directly.
 
+### On MacOS, using ZSH
+
+To add kubectl autocomplete to your ZSH profile:
+
+```shell
+sed "s/^\(plugins=(\)\(.*\)\()\)$/\1\2 kubectl\3/" ~/.zshrc
```
Comment (ahmetb) [inline docs/tasks/tools/install-kubectl.md]: I'm -1 for this. It's a very opaque command most users don't understand. First of all, what happens if there's no `plugins=` in .zshrc.

I also have a question: Have you tested this without oh-my-zsh? I believe plugins are an oh-my-zsh concept and not `zsh` so I've reason to believe this command assumes oh-my-zsh.

Comment (zoidbergwill) [inline docs/tasks/tools/install-kubectl.md]: Adding this to `~/.zshrc` is cleaner, I think:

```
 source <(kubectl completion zsh)
```

Comment (ahmetb) [inline docs/tasks/tools/install-kubectl.md]: Agreed, seems to be working pretty much the same way.

**Comment by chenopis** [comment]
@randavidovitz FYI, there seems to be a merge conflict now.

**Comment by steveperry-53** [comment]
ping @randavidovitz, Can you resolve the merge conflict?

**Comment by jaredbhatti** [comment]
One more ping.  Can you resolve?

**Comment by zacharysarah** [comment]
@randavidovitz Please feel free to re-open this PR when you're ready to resolve the merge conflict. Thanks!

**Pull request closed without merging**